etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/hapi/HapiResultTest.groovy

```diff
@@ -1,16 +1,30 @@
 package gov.hhs.cdc.trustedintermediary.external.hapi
 
-import org.hl7.fhir.r4.model.Bundle
+import gov.hhs.cdc.trustedintermediary.context.TestApplicationContext
+import gov.hhs.cdc.trustedintermediary.etor.messages.MessageHdDataType
+import gov.hhs.cdc.trustedintermediary.wrappers.HapiFhir
+import org.hl7.fhir.r4.model.*
```

Comment:
Reminder to not use wildcard imports. You should be able to change this setting in your IDE to avoid it

Reply:
Good catch. I hope everyone is using the githooks.

Reply:
fixed.